sln/WebApiOData.AspNetCore.sln

```diff
@@ -19,6 +19,8 @@ Project("{D954291E-2A0B-460D-934E-DC6B0785DB48}") = "Microsoft.AspNet.OData.Test
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.AspNetCore.OData.Test", "..\test\UnitTest\Microsoft.AspNetCore.OData.Test\Microsoft.AspNetCore.OData.Test.csproj", "{2B8085F7-3625-489E-A963-E29C2BD1AA76}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AspNetCoreODataSample.DynamicModels.Web", "..\samples\AspNetCoreODataSample.DynamicModels.Web\AspNetCoreODataSample.DynamicModels.Web.csproj", "{DC9202F7-ED44-4E40-A509-4C4BE01548B3}"
```

Comment:
@Danielku15 We have a separate repository for the samples. It is OData/Samples. The PR needs to have the code and the tests for the WebAPI repo.

Reply:
Sure, for the final version I plan to move it over to the samples. I thought of including it first in this PR for manually testing the feature and also for illustrating the change for the review. I thought otherwise quickly the question arises: "why is this needed? how is this meant to be used?". I mentioned this in the PR task list: " Sample project with a user-defined model system added (**likely needs to go to OData/ODataSamples for final merge**)
"